apps/agile-project-fe/src/app/components/button/button.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import styled, { css } from 'styled-components';
 
 export interface ButtonProps {
-  variant: 'confirm' | 'cancel' | 'neutral' | 'warning';
+  variant: 'confirm' | 'cancel' | 'neutral' | 'warning' | string;
```

Comment:
If we allow the `string` type here, then the component will accept absolutely **any** string, but we want it to only accept the values we have defined for `variant`